node/src/GlideClient.ts

```diff
@@ -167,14 +168,19 @@ export class GlideClient extends BaseClient {
      *   See https://redis.io/topics/Transactions/ for details on Redis Transactions.
      *
      * @param transaction - A Transaction object containing a list of commands to be executed.
+     * @param decoder - An optional parameter to decode all commands in the transaction. If not set, 'Decoder.String' will be used.
      * @returns A list of results corresponding to the execution of each command in the transaction.
      *      If a command returns a value, it will be included in the list. If a command doesn't return a value,
      *      the list entry will be null.
      *      If the transaction failed due to a WATCH command, `exec` will return `null`.
      */
-    public exec(transaction: Transaction): Promise<ReturnType[] | null> {
+    public exec(
+        transaction: Transaction,
+        decoder: Decoder = Decoder.String,
```

Comment:
Why not
```suggestion
        decoder?: Decoder,
```
We could make default decoder configurable later.

Reply:
Does not matter, don't hardcode default decoder please. Align this with other API.